doc/man3/pbs_preempt_jobs.3B

```diff
@@ -0,0 +1,129 @@
+.\" Copyright (C) 1994-2019 Altair Engineering, Inc.
+.\" For more information, contact Altair at www.altair.com.
+.\"
+.\" This file is part of the PBS Professional ("PBS Pro") software.
+.\"
+.\" Open Source License Information:
+.\"
+.\" PBS Pro is free software. You can redistribute it and/or modify it under the
+.\" terms of the GNU Affero General Public License as published by the Free
+.\" Software Foundation, either version 3 of the License, or (at your option) any
+.\" later version.
+.\"
+.\" PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+.\" WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+.\" FOR A PARTICULAR PURPOSE.
+.\" See the GNU Affero General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU Affero General Public License
+.\" along with this program.  If not, see <http://www.gnu.org/licenses/>.
+.\"
+.\" Commercial License Information:
+.\"
+.\" For a copy of the commercial license terms and conditions,
+.\" go to: (http://www.pbspro.com/UserArea/agreement.html)
+.\" or contact the Altair Legal Department.
+.\"
+.\" Altair’s dual-license business model allows companies, individuals, and
+.\" organizations to create proprietary derivative works of PBS Pro and
+.\" distribute them - whether embedded or bundled with other software -
+.\" under a commercial license agreement.
+.\"
+.\" Use of Altair’s trademarks, including but not limited to "PBS™",
+.\" "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+.\" trademark licensing policies.
+.\"
+
+.TH pbs_preempt_jobs 3B "15 November 2019" Local "PBS Professional"
+.SH NAME
+.B pbs_preempt_jobs 
+\- preempt a list of jobs
+.SH SYNOPSIS
+#include <pbs_error.h>
+.br
+#include <pbs_ifl.h>
+.sp
+.nf
+.B int preempt_job_info *pbs_preempt_jobs(int connect, char **jobID_list)
+.fi
+.SH DESCRIPTION
+Sends the server a list of jobs to be preempted.
+
+Sends a 
+.I Preempt Jobs
+(93) batch request to the server over the connection specified by 
+.I connect.
+
+Returns a list of preempted jobs along with the method used to preempt
+each one.
+
+.SH ARGUMENTS
+.IP connect 8
+Return value of 
+.B pbs_connect().  
+Specifies connection handle over which to send batch request to server.
+
+.IP jobID_list 8
+List of job IDs to be preempted, as a null-terminated array of pointers to strings.  
+.br
+Format for a job ID:
+.br
+.I <sequence number>.<server name>
+.br
+Format for a job array ID:
+.br
+.I <sequence number>[].<server name>
+.br
+For example:
+.RS 11
+.nf
+const char *joblist[3];
+joblist[0]="123@myserver";
+joblist[1]="456@myserver";
```

Comment:
Right above you say <sequence number>.<server name> and here you are using an @.